### PR TITLE
removed assetic

### PR DESCRIPTION
**Pull request by peterprochaska**
## Description
Removed Assetic because of some security issues

## Motivation and Context
Make owncloud more secure

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: .htaccess [hunk 1]
```diff
@@ -74,3 +74,7 @@ Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
 </IfModule>
+#### DO NOT CHANGE ANYTHING ABOVE THIS LINE ####
```
Comment (DeepDiver1975) [inline .htaccess]: please do not commit htaccess - thx

**Review by DeepDiver1975 — CHANGES_REQUESTED** [review]

**Comment by DeepDiver1975** [comment]
if i remember correctly there is a repairstep related to assets - maybe a occ command as well? THX

**Comment by PVince81** [comment]
How dangerous to put this in that close to the release ?

In the future it would be nice to address library changes at the beginning of a milestone.

**Comment by DeepDiver1975** [comment]
> In the future it would be nice to address library changes at the beginning of a milestone.

sure - but sec scanner revealed this last week ....

**Comment by DeepDiver1975** [comment]
@Peter-Prochaska composer.lokc also need changes afaik

**Comment by PVince81** [comment]
Ok, then please get this reviewed and merged before this evening.

The current code changes look fine.

**Comment by PVince81** [comment]
If this is really important, please finish this as soon as possible. Thanks.

**Comment by PVince81** [comment]
@Peter-Prochaska please always ping after updating so I can get a notification that this is ready again, else it gets lost forever in the dark abyss of ticket overflow

**Comment by PVince81** [comment]
I setup OC and it still seems to work fine 👍

**Comment by PVince81** [comment]
rebased for CI... mysql mb4 docker times out again in multiple PRs...

**Comment by Peter-Prochaska** [comment]
> it gets lost forever in the dark abyss of ticket overflow
wow ;-)

**Comment by PVince81** [comment]
some weird dependency issues on Travis: https://travis-ci.org/owncloud/core/jobs/224313100

```
php build/composer.phar install --dev

You are using the deprecated option "dev". Dev packages are installed by default now.

Loading composer repositories with package information

Installing dependencies (including require-dev) from lock file

Your requirements could not be resolved to an installable set of packages.

  Problem 1

    - Installation request for ocramius/package-versions 1.1.2 -> satisfiable by ocramius/package-versions[1.1.2].

    - ocramius/package-versions 1.1.2 requires php ~7.0 -> your PHP version (5.6.5) does not satisfy that requirement.

  Problem 2

    - Installation request for ocramius/proxy-manager 2.0.4 -> satisfiable by ocramius/proxy-manager[2.0.4].

    - ocramius/proxy-manager 2.0.4 requires php 7.0.0 - 7.0.5 || ^7.0.7 -> your PHP version (5.6.5) does not satisfy that requirement.

  Problem 3

    - ocramius/proxy-manager 2.0.4 requires php 7.0.0 - 7.0.5 || ^7.0.7 -> your PHP version (5.6.5) does not satisfy that requirement.

    - sensiolabs/behat-page-object-extension v2.0.1 requires ocramius/proxy-manager ^1.0||^2.0 -> satisfiable by ocramius/proxy-manager[2.0.4].

    - Installation request for sensiolabs/behat-page-object-extension v2.0.1 -> satisfiable by sensiolabs/behat-page-object-extension[v2.0.1].

make: *** [lib/composer/phpunit] Error 2
```

File: .htaccess [hunk 2]
```diff
@@ -73,4 +73,4 @@ AddDefaultCharset utf-8
 Options -Indexes
 <IfModule pagespeed_module>
   ModPagespeed Off
-</IfModule>
+</IfModule>
```
Comment (PVince81) [inline .htaccess]: Next time please don't modify random files, thanks.

Regression: https://github.com/owncloud/core/commit/dbb37e7684e9d6e6d6618e5dcb66ba5a0192c3da

There is core that appends line in htaccess and it creates a syntax error if the newline is missing.

**Comment by kdslkdsaldsal** [comment]
FWIW is stuff is removed / dropped it would make sense to create an issue in the documentation tracker to remove all related stuff: https://github.com/owncloud/documentation/issues/3438

**Pull request merged**